source/_components/plex.markdown

```diff
@@ -46,8 +37,20 @@ You can also enable the plex platform directly by adding the following lines to
 
 ```yaml
 # Example configuration.yaml entry
-media_player:
-  - platform: plex
+plex:
+```
+
+```yaml
```

Comment:
Please move this additional example to a separate section below the configuration options list.